gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/Form.java

```diff
@@ -36,6 +37,15 @@
  */
 public class Form extends AbstractForm implements HasType<FormType> {
 
+    public Form() {
```

Comment:
We need the empty constructor otherwise you must provide the FormType, and this will break the API.